tests/io/test_editor.py

```diff
@@ -170,3 +171,46 @@ def test_editor_details(api):
     for each_scan_id in scan_ids_list:
         api.scans.delete(each_scan_id)
 
+
+@pytest.mark.vcr()
+def test_editor_parse_plugins_id_keyerror(api):
```

Comment:
Testcase name should be `test_editor_parse_plugins_id_keyerror` or `test_editor_parse_plugins_family_id_keyerror` ?